source/_integrations/http.markdown

```diff
@@ -116,6 +116,9 @@ http:
     - 10.0.0.200      # Add the IP address of the proxy server
     - 172.30.33.0/24  # You may also provide the subnet mask
 ```
+Note: The `172.30.33.0/24` line is not an example but a mandatory line.
+
+Remember to enable WebSockets headers in your reverse proxy configuration.
```

Comment:
This depends on which reverse proxy you are using. Some handle this automatically, other require specific rulesets to upgrade the connection.

Configuring your reverse proxy is considered out of scope for Home Assistant.

Reply:
If not here, how should the doc set that HA needs the websockets headers to work? its not a configuration but a requirement or I am missing something?

Reply:
You are not missing something, however, how to configure a reverse proxy itself, is out of scope for Home Assistant.

Documentation around configuration a reverse proxy itself has been removed from the Home Assistant documentation in the past. We are not keen on re-introducing it again.

Configuring a reverse proxy is an advanced subject, for which we consider the user capable of knowing how it works and what to do.